### PR TITLE
Add external links to supported libraries

### PR DESCRIPTION
**Pull request by sleipnir**
Adding links following the pattern of the new plugin for Paradox

### CODE REVIEW

File: docs/src/main/paradox/user/lang/index.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Cloudstate user functions can be implemented in any language that supports gRPC.
 * [Go](go/index.md)
 * [Kotlin](kotlin/index.md)
 * [Dart](dart/index.md)
+* [Springboot](springboot/index.md)
```
Comment (pvlugter) [inline docs/src/main/paradox/user/lang/index.md:13]: Missing the .Net page:

```suggestion
* [Spring Boot](springboot/index.md)
* [.Net](dotnet/index.md)
```

**Comment by pvlugter** [comment]
@sleipnir, you may want to base on an updated master branch, to avoid all the extra commits in the PR

**Comment by sleipnir** [comment]
> @sleipnir, you may want to base on an updated master branch, to avoid all the extra commits in the PR

Yes @pvlugter , that was a distraction of mine and I apologize for that

**Comment by sleipnir** [comment]
Thanks @pvlugter

**Pull request merged**